packages/shopify-app-remix/docs/generated/generated_static_pages.json

```diff
@@ -0,0 +1,135 @@
+[
+  {
+    "id": "shopify-app-remix",
+    "title": "Shopify App package for Remix",
+    "description": "The @shopify/shopify-app-remix package enables Remix apps to authenticate with Shopify and make API calls. It uses [App Bridge](/docs/api/app-bridge-library) to enable apps to embed themselves in the Shopify Admin.",
+    "sections": [
+      {
+        "type": "Generic",
+        "anchorLink": "install",
+        "title": "Install",
+        "sectionContent": "The quickest way to create a new app is using the Shopify CLI. You can follow the getting started guide to create a new app that uses this package.\n\nYou can follow the instructions on this page to set up an existing app to use this package. Start by installing it using your preferred package manager.",
+        "sectionCard": [
+          {
+            "name": "Build an app",
+            "subtitle": "Navigate to",
+            "url": "/docs/apps/getting-started/build-qr-code-app",
+            "type": "tutorial"
+          }
+        ],
+        "codeblock": {
+          "title": "Install package",
+          "tabs": [
+            {
+              "title": "npm",
+              "code": "npm i --save @shopify/shopify-app-remix\n"
+            },
+            {
+              "title": "yarn",
+              "code": "yarn add @shopify/shopify-app-remix\n"
+            },
+            {
+              "title": "pnpm",
+              "code": "pnpm add @shopify/shopify-app-remix\n"
+            }
+          ]
+        }
+      },
+      {
+        "type": "Generic",
+        "anchorLink": "backend",
+        "title": "Set up your backend",
```

Comment:
Should we split this into two steps 1) Create the Shopify Object, 2) create an Auth route. Then we would have room to add a few more details to each.